src/pages/Home.page.vue

```diff
@@ -66,10 +75,20 @@ const { t } = useI18n();
         <div v-if="toolStore.favoriteTools.length > 0">
           <h3 class="mb-5px mt-25px font-500 text-neutral-400">
             {{ $t('home.categories.favoriteTools') }}
+            <c-tooltip tooltip="Drag and drop to reorder favorites">
```

Comment:
Put this text line in en locale file please 
```suggestion
            <c-tooltip :tooltip="$t('...')">
```

Reply:
good point 👍, rebased changes to original commit to include:
- ```<c-tooltip :tooltip="$t('home.categories.favoritesDndToolTip')">```
- updated all available locales as well